src/PackageCompiler.jl

```diff
@@ -183,7 +183,7 @@ function run_precompilation_script(project::String, sysimg::String, precompile_f
     cmd = `$(get_julia_cmd()) --sysimage=$(sysimg) --project=$project
             --compile=all --trace-compile=$tracefile $arg`
     @debug "run_precompilation_script: running $cmd"
-    read(cmd)
+    run(cmd)  # `Run` this command so that we'll display stdout from the user's script.
```

Comment:
```suggestion
    @info "===== Start precompile execution ====="
    run(cmd)  # `Run` this command so that we'll display stdout from the user's script.
    @info "===== End precompile execution ====="
```

Reply:
@KristofferC: Does this look like what you had in mind? :)

Thanks!